packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagelist/_index.scss

```diff
@@ -9,6 +9,14 @@
     flex-direction: column;
     width: 100%;
   }
+
+  .dx-scrollable-native {
+    &.dx-scrollable-native-ios{
```

Comment:
```suggestion
    &.dx-scrollable-native-ios {
```

Reply:
If there are no other remarks, I'll fix this typo in the next PR to avoid rerunning the tests.